### ISSUE TITLE
Build error with new verion

### ISSUE THREAD

**Issue by igoriok1994** [taye/interact.js#814]
Interact.js version 1.9.12

Build error from UglifyJS.

```
ERROR in npm.interactjs.313dd53e.js from UglifyJs
Unexpected token: operator «>» [./node_modules/interactjs/dist/interact.min.js:2,308][npm.interactjs.313dd53e.js:7,94]
```

**Comment by taye** [comment]
Thanks for reporting. I released `v1.9.13` with a fix for this.

**Comment by igoriok1994** [comment]
> Thanks for reporting. I released `v1.9.13` with a fix for this.

Thank you.
